CMakeLists.txt

```diff
@@ -1,59 +1,79 @@
-cmake_minimum_required(VERSION 3.8.2)
+cmake_minimum_required(VERSION 3.15)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
 
 project(
   topgg
+  VERSION 1.0.0
   LANGUAGES CXX
   HOMEPAGE_URL "https://docs.top.gg/docs"
   DESCRIPTION "The official C++ wrapper for the Top.gg API."
 )
 
-set(CMAKE_BUILD_TYPE Debug CACHE STRING "Build type")
-option(BUILD_SHARED_LIBS "Build shared libraries" ON)
-option(ENABLE_CORO "Support for C++20 coroutines" OFF)
```

Comment:
Why did you remove `ENABLE_CORO`?

Reply:
This option lets users of this library choose whether to utilize C++20 coroutines or not (done through callbacks, but compatible with C++17).